grails-app/controllers/au/org/ala/biocollect/merit/ProjectController.groovy

```diff
@@ -826,6 +854,11 @@ class ProjectController {
         }
 
         if (trimmedParams.isUserPage) {
+            boolean isAlaAdmin = userService.userIsAlaAdmin()
+
+            if (isAlaAdmin)
```

Comment:
It looks to me ALA admin can only see published projects in their user page. is the logic correct?

Reply:
The code needs to satisfy the following conditions
1. ALA admin can see published and unpublished projects on project finder.
2. Regular users should only see published projects only.
3. Regular users when viewing `My projects` should be able to see their projects that are both published and unpublished.